site/en/docs/webstore/images/index.md

```diff
@@ -66,9 +67,13 @@ Here are some icons that follow these guidelines.
   </tr>
 </table>
 
-<div class="aside aside--note"><b>Note</b>: If you upload an image that has no alpha, it will be placed in a frame with rounded corners (12-pixel corner radius).</div>
+{% Aside 'note' %}
 
-### Icon size
+**Note:** If you upload an image that has no alpha, it will be placed in a frame with rounded corners (12-pixel corner radius).
```

Comment:
I prefer to skip the text that says Note: because I think the CSS note-ness is enough. I might be persuaded on accessibility grounds.